templates/zerver/emails/followup_day1.source.html

```diff
@@ -25,7 +25,7 @@
         <li>{{ _('Use your LDAP account to log in') }}<br></li>
         {% endif %}
     {% else %}
-        <li>{{ _('Email:') }} {{ email }}<br></li>
+        <li>{% trans email=macros.email_tag(email) %}Email: {{ email }}{% endtrans %}<br></li>
```

Comment:
Do we actually want this to be a `mailto` tag?  I feel like it might be better framing this as data, not something to click.

Reply:
Yeah. We can remove that. I think the reason why we skipped this in the first place was most likely to avoid the email tag in translation.